Add support for metadata in CBS (#4581)
kyma-project/kyma: docs/headless-cms/06-01-docstopic.md

| **spec.sources.type** |    **YES**   | Specifies the type of assets included in the DocsTopic CR. |
| **spec.sources.name** |    **YES**   | Defines an identifier of a given asset. It must be unique if there is more than one asset of a given type in a DocsTopic CR. |
| **spec.sources.mode** |    **YES**   | Specifies if the asset consists of one file or a set of compressed files in the ZIP or TAR format. Use `single` for one file and `package` for a set of files.  |

> | **spec.sources.metadata** |    **NO**   | Specifies a set of metadata for the Asset. Define it in a valid YAML or JSON format. |

Reviewer: in each doc where we describe **spec.sources.metadata** specify for the user that one of the possible use case for metadata is to use is as a contract between source and the presentation layer, to enable UI to decide on a  source level, if something specific has to be done/modified in the source before it is rendered for the user

> | **spec.sources.metadata** |    **NO**   | Specifies a set of metadata for the Asset. Define it in a valid YAML or JSON format. |

Reviewer 2: ```suggestion
| **spec.sources.metadata** |    **NO**   | Specifies a set of metadata for the Asset that you want to render, disable, or modify in the UI. Define it in a valid YAML or JSON format. |
```

Reviewer 2: and pls copy it to every doc when this parameter is described ;)

> | **spec.sources.metadata** |    **NO**   | Specifies the set of the metadata for Asset. It must be defined in a valid YAML/JSON format. |

Reviewer 2: ```suggestion
| **spec.sources.metadata** |    **NO**   | Specifies a set of metadata for the Asset. Define it in a valid YAML or JSON format. |
```
